strawberryfields/apps/subgraph.py

```diff
@@ -80,6 +102,9 @@ def search(
         min_size (int): minimum size to search for dense subgraphs
         max_size (int): maximum size to search for dense subgraphs
         max_count (int): maximum number of densest subgraphs to keep track of for each size
+        node_select (str, list or array): method of settling ties when more than one node of
```

Comment:
The term "settling ties" is a bit confusing to me. Method of settling ties can be a numpy array or a list?

Reply:
Maybe good to say that the numpy array or list specifies the node weights